src/legacy/ui/ui_render/bootstrap/template.js.hbs

```diff
@@ -3,6 +3,34 @@ window.__kbnStrictCsp__ = kbnCsp.strictCsp;
 window.__kbnDarkMode__ = {{darkMode}};
 window.__kbnThemeVersion__ = "{{themeVersion}}";
 window.__kbnPublicPath__ = {{publicPathMap}};
+window.__kbnBundles__ = (function () {
+  var modules = {};
+
+  function has(prop) {
```

Comment:
why not use Map?

Reply:
Because `get` calls require as needed, which is what allows circular deps to work.

Reply:
Also, this code executes before polyfills are loaded.

Reply:
> Also, this code executes before polyfills are loaded.

ok, just to note that we don't support IE11 anymore, so shouldn't be a problem